### PR TITLE
SMS add response to user replies

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@gsvr ReViEw PlEaSe

File: test/fixtures_public.js [hunk 1]
```diff
@@ -7,6 +7,7 @@
 // 05059996666: registered user - mother that receives own messages, linked to 05059997777
 // 05059997777: registered user - family member that receives household messages for 05059996666
 // 05059998888: number being changed to
+// 05059999999: number used to test sms app; does unsubscribe
```
Comment (gsvr) [inline test/fixtures_public.js]: Let's not add another number.  Just use ..2222 again.

**Comment by gsvr** [comment]
@NicolaasDieBaas First pass done.

File: test/fixtures_public.js (hunk 1 above)
Comment (NicolaasDieBaas) [inline test/fixtures_public.js]: doing an opt-out require a GET with a subscription url so i added a new number for this reason... i guess I could add a subscription-GET-request to ..22222... ?

Comment (gsvr) [inline test/fixtures_public.js]: Ah, sorry I've done this in an issue that hasn't landed yet.  Please look in #55 if you can find an applicable piece of code to use.

**Comment by NicolaasDieBaas** [comment]
@gsvr plz review.  You'll note that that one fixture is still in there... utils.subscription_unsubscribe_all does a GET request with a subscription url using msisdn as parameter

**Comment by NicolaasDieBaas** [comment]
@gsvr would like to discuss how opting-out via sms needs to look like after all the recent work done on opt-out functionality for ussd/voice public...

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**